framework/wazuh/core/config/models/ssl_config.py

```diff
@@ -86,7 +87,7 @@ class IndexerSSLConfig(WazuhConfigBaseModel, ValidateFilePathMixin):
     certificate: str = ''
     certificate_authorities: List[str] = Field(default=[''], min_length=1, exclude=True)
     verify_certificates: bool = True
-    certificate_authorities_bundle: str = WAZUH_INDEXER_CA_BUNDLE
+    certificate_authorities_bundle: Path = WAZUH_INDEXER_CA_BUNDLE
```

Comment:
This fixes a warning (`Expected str but got PosixPath...`) that had no functional impact